### PR TITLE
Michael LIttleton - DOM II 

### PR DESCRIPTION
**Pull request by mglittleton**
@clarakosi

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -1 +1,190 @@
...
+const destinations = document.querySelectorAll(".destination .btn");
+const destinationDivs = document.querySelectorAll(".destination");
+const centerTitleTop = document.querySelector(".content-section h2");
+const centerTitleBottom = document.querySelector(".inverse-content h2");
+const topPic = document.querySelector(".intro img");
+const topTitle = document.querySelector(".logo-heading");
+
+// 1 & 2 navbar borders pop in with mouseover and return to normal with mouseleave
+const borderBoxOn = function() {
+  this.style.border = "1px solid black";
```
Comment (clarakosi) [inline js/index.js]: Great job structuring this! By creating the functions separately you are able to reuse them.

File: js/index.js [hunk 2]
```diff
@@ -1 +1,190 @@
...
+    switchDir = -1;
+  }
+  size += switchDir * 10;
+};
+
+lastPic.addEventListener("wheel", increaseSize);
+
+// 4 each individual destination paragraph increases width on click of the buttons - demonstrates bubbling fix
+const bumpOut = function() {
+  event.stopPropagation();
```
Comment (clarakosi) [inline js/index.js]: Great use of stopPropagation here!

File: js/index.js [hunk 3]
```diff
@@ -1 +1,190 @@
...
+  centerTitleTop.style.opacity = "1";
+  tempString = centerTitleBottom.innerHTML;
+  centerTitleBottom.innerHTML = centerTitleTop.innerHTML;
+  centerTitleTop.innerHTML = tempString;
+};
+
+document.addEventListener("drop", dragSwap);
+
+// 10 & 11 flips the header image every 50ms on page load until double-click
+// TODO need to figure out why the image is overriding the navbar
```
Comment (clarakosi) [inline js/index.js]: Still need help here?

Comment (mglittleton) [reply js/index.js]: Nope. Thanks. I fixed it by changing the z-index of the navbar to 2

**Pull request closed without merging**